doc/.rtd-environment.yml

```diff
@@ -27,7 +27,7 @@ dependencies:
       - stsci.imagestats
       - stsci.skypac
       - stsci.stimage
-      - "tweakwcs>=0.4.0"
+      - "tweakwcs>=0.7.0"
       - stregion
```

Comment:
Does this setting for tweakwcs reconcile with setup.py?  I am not sure the order of evaluation.

Reply:
Actually, that would not have caused problems since the installation using 'setup.py' would have forced a newer version (0.7.2) to be installed.  However, this has been updated to be consistent with `setup.py` in 8f33c83 just to prevent any problems.